cartridge-cli.lua

```diff
@@ -1389,10 +1448,22 @@ local function build_application(dir)
         -- deleting files matching patterns from .cartridge.ignore
         info('Remove files matching patterns from %s', DEP_IGNORE_FILE_NAME)
         remove_ignored(dir)
-        remove_by_path(fio.pathjoin(dir, DEP_IGNORE_FILE_NAME))
-        remove_by_path(fio.pathjoin(dir, DEP_PREBUILD_SCRIPT_NAME))
-        info('Remove .git directory')
-        remove_by_path(fio.pathjoin(dir, '.git'))
+
+        -- remove special files
+        for _, filename in ipairs({DEP_IGNORE_FILE_NAME, DEP_PREBUILD_SCRIPT_NAME}) do
+            local filepath = fio.pathjoin(dir, filename)
+            if fio.path.exists(filepath) then
+                info('Remove %s', filename)
```

Comment:
Could you please provide an example of log of packaging some simple module?

I suppose that such verbosity could be redundant. And this messages should have "verbose" log level

Reply:
```

Reply:
It's just two messages. I think it's better than deleting these files without any message.

Reply:
`.../.cartridge/tmp/cartridge-build-8f7e3cefe056829c`

Why '...' at start?

In general ok